PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install JumpCloud Password Manager App.md

```diff
@@ -11,68 +11,70 @@ windows
 ```
 # Set $LaunchPasswordManager to $false  ON LINE 35 if you do not wish to launch the password manger after installation
 
+# Get the current logged on User
+$loggedUser = Get-WmiObject -Class Win32_ComputerSystem | Select-Object -ExpandProperty UserName
+$loggedUser = $loggedUser -replace '.*\\'
+
+# Construct the Registry path using the user's SID
+$userSID = (New-Object System.Security.Principal.NTAccount($loggedUser)).Translate([System.Security.Principal.SecurityIdentifier]).Value
+$registryPath = "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion\ProfileList\$userSID"
+
+# Get the ProfileImagePath value from the Registry
+$loggedOnUserProfileImagePath = Get-ItemPropertyValue -Path $registryPath -Name 'ProfileImagePath'
+Write-Output "loggedOnUserProfileImagePath: $loggedOnUserProfileImagePath"
```

Comment:
my only suggestion here is to print out some human readable bit here like " Logged On User Profile Path" vs "loggedOnUserProfileImagePath"